test/s2wasm_known_binaryen_shell_test_failures.txt

```diff
@@ -28,7 +28,6 @@ bitfld-5.c.s.wast
 memcpy-1.c.s.wast
 mode-dependent-address.c.s.wast
 multi-ix.c.s.wast
-pr29006.c.s.wast
```

Comment:
Don't know? The only relevant change was the size of the memory, so maybe there's an off-by-one error somewhere in the binaryen interpreter  or something that gets papered over because of the larger memory? (or in LLVM, but that seems less likely because this test works in V8)